config/file_config.go

```diff
@@ -550,6 +550,22 @@ func (f *fileConfig) GetCollectorType() (string, error) {
 	return f.conf.Collector, nil
 }
 
+func (f *fileConfig) GetRunningConfig() (map[string]interface{}, error) {
+	configs := make(map[string]interface{})
+	keys := f.config.AllKeys()
+	for _, key := range keys {
+		configs[key] = f.config.GetString(key)
+		// mask sensitive strings
+		maskKeys := []string{"key", "password", "token"}
```

Comment:
I agree, as the endpoint is authenticated and requires direct access to the cluster, I think it would be okay to not mask those values.

However, if you do decide to mask these values, you could use a `switch key.ToLower()` in place of creating an array and looping over it.

Reply:
I wrote it that way first but it isn't a perfect match so I'm not sure what goes in the 'case' to handle "contains".